javascript-bank/bank.js

```diff
@@ -0,0 +1,39 @@
+/* exported Bank */
+
+class Bank {
+  constructor() {
+    this.nextAccountNumber = 1;
+    this.accounts = [];
+    this.openAccount = this.openAccount.bind(this);
+    this.getAccount = this.getAccount.bind(this);
+    this.getTotalAssets = this.getTotalAssets.bind(this);
```

Comment:
You don't need to bind these since we are not passing them as callbacks anywhere.